src/codegen/type/bigint_type.cpp

```diff
@@ -464,6 +370,127 @@ struct Modulo : public TypeSystem::BinaryOperatorHandleNull {
   }
 };
 
+////////////////////////////////////////////////////////////////////////////////
+///
+/// Unary operations
+///
+////////////////////////////////////////////////////////////////////////////////
+
+// Abs
```

Comment:
Any reason for moving all these functions? Convention for all other types is to put all unary functions above the binary ones.